### ISSUE TITLE
Pasted melodies are quantized

### ISSUE THREAD

**Issue by Spekular** [LMMS/lmms#4056]
Tested on RC4, Windows 10 64bit.

Copying a melody and pasting it into piano roll will cause it to be quantized at the current quantization level. For example:
![image](https://user-images.githubusercontent.com/8918426/34070682-9976dfac-e26a-11e7-88e2-f55efdae2433.png)

I believe the melody should be preserved as-is. If a user wants the melody to be quantized they can use the quantize button.

**Comment by zonkmachine** [comment]
I agree. This behaviour is in 1.1.3 too.

Code here: https://github.com/LMMS/lmms/blob/stable-1.2/src/gui/editors/PianoRoll.cpp#L3766

~Fixed by:

```cpp
diff --git a/src/gui/editors/PianoRoll.cpp b/src/gui/editors/PianoRoll.cpp
index ae5c33c..b283ca0 100644
--- a/src/gui/editors/PianoRoll.cpp
+++ b/src/gui/editors/PianoRoll.cpp
@@ -3763,7 +3763,7 @@ void PianoRoll::pasteNotes()
                        cur_note.setSelected( true );
 
                        // add to pattern
-                       m_pattern->addNote( cur_note );
+                       m_pattern->addNote( cur_note, false );
                }
 
                // we only have to do the following lines if we pasted at
```

**Comment by zonkmachine** [comment]
OK. When I saw the title of this issue I just assumed that it was a bug from a commit with my name on it. It wasn't but it was easy enough to fix so I hacked that in #4058 . I don't know what a proper issue label is but think fixing it in 1.2 is appropriate.
